pkgs/applications/office/kitsas/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, mkDerivation, fetchFromGitHub, qmake, qtsvg, qtcreator, libsForQt5, libzip, pkg-config  }:
+
+mkDerivation rec {
+  pname = "kitsas";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "artoh";
+    repo = "kitupiikki";
+    rev = "v" + version;
+    sha256 = "1qac6cxkb45rs5pschsf2rvpa789g27shmrwpshwahqzhw42xvgl";
+  };
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ qmake qtsvg libsForQt5.poppler libzip ];
+
+  phases = [ "unpackPhase" "configurePhase" "buildPhase" "installPhase" "fixupPhase" ];
+
+  # We use a separate build-dir as otherwise ld seems to get confused between
+  # directory and executable name on buildPhase.
+  configurePhase = ''
```

Comment:
Can you convert this into preConfigure and qmakeFlags?

Reply:
This was a bit more difficult change for me, but I think I got it made now. Thanks for suggesting this one (I think some new "clicks" happened in my head now - small but anyway).